compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -3211,6 +3211,101 @@ TR_S390Peephole::AGIReduction()
 
    return performed;
    }
+
+/* For runs with -Xgc:concurrentScavenge on hardware that doesn't support guarded storage facility,
+ * swap LGG/LLGFSG with regular loads and software read barrier sequence that checks if concurrent scavange is in progress
+ * and if the current object pointer is in the evacuate space then calls the GC helper to update the object pointer.
+ */
+
+bool
+TR_S390Peephole::SoftwareReadBarrier()
```

Comment:
Other function names in this class that don't begin with an acronym begin with a lower-case letter.  Please make this consistent.

Reply:
Changed the function name to replaceGuardedLoadWithSoftwareReadBarrier.